### PR TITLE
RJMR-267: move and update repo

### PR DESCRIPTION
**Pull request by FilippoRezzonico**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Docker Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.BOT_DOCKER_USERNAME }}
```
Comment (fredmaggiowski) [inline .github/workflows/test.yml:70]: @davidebianchi can you check if the name of the variable is correct?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
+
+- [RJMR-267](https://makeitapp.atlassian.net/browse/RJMR-267): added makefile and github actions. Updated dependencies.
```
Comment (fredmaggiowski) [inline CHANGELOG.md]:
```suggestion
- added makefile and github actions. Updated dependencies.
```

no need for internal tracking logs; also I'd add the releaser action and deprecate the changelog (like we did [here](https://github.com/rond-authz/rond/blob/main/.github/release.yml))

Comment (FilippoRezzonico) [reply CHANGELOG.md]: Removed entry from changelog and added deprecation warning in it. As suggested, releaser action has been added

File: .github/workflows/test.yml (hunk 1 above)
Comment (davidebianchi) [reply .github/workflows/test.yml:70]: Yes, they are correct

**Pull request merged**